AUTHORS.rst

```diff
@@ -29,3 +29,4 @@ UI for Invenio-Search.
 
 - CERN <info@invenio-software.org>
 - Jiri Kuncar <jiri.kuncar@cern.ch>
+- Nikos Filippakis <nikolaos.filippakis@cern.ch>
```

Comment:
Can I get you to fix the author list in this PR as well, since I can see it's outdated. The following people are missing:

Esteban J. G. Gabancho <esteban.gabancho@gmail.com>
Harris Tzovanakis <me@drjova.com>
Jiri Kuncar <jiri.kuncar@cern.ch>
Lars Holm Nielsen <lars.holm.nielsen@cern.ch>
Leonardo Rossi <leonardo.r@cern.ch>